packages/td-bank-integration/src/td-online-mart-payment-strategy.ts

```diff
@@ -147,4 +160,29 @@ export default class TDOnlineMartPaymentStrategy implements PaymentStrategy {
 
         return this.tdOnlineMartClient;
     }
+
+    private async processWithAdditionalAction(error: unknown): Promise<void> {
+        if (!isTdOnlineMartAdditionalAction(error)) {
+            throw error;
+        }
+
+        const { three_ds_result: threeDSResult }: TdOnlineMartThreeDSErrorBody = error.body;
+        const { formUrl, threeDSSessionData, creq } = threeDSResult || {};
+
+        if (!formUrl || !threeDSSessionData || !creq) {
+            throw new PaymentArgumentInvalidError(['formUrl', 'threeDSSessionData', 'sreq']);
```

Comment:
```suggestion
            throw new PaymentArgumentInvalidError(['formUrl', 'threeDSSessionData', 'creq']);
```

Reply:
Also, I don't really sure that it is right error to show for the customer, because the message is: 
"Make sure the following fields are provided correctly:"
I don't think that user can fix that on their end.. So it's better to do some search and throw another message.

What do you think?

Reply:
In this case users can't fix this issue on their end. Because these options are broadcast from TDBank over BigPay, and the user can’t change its values. But understanding what goes wrong for a shopper, by simple screenshot for support, will help us more accurately and faster understand and fix bugs.